.github/readme/partials/license.md

```diff
@@ -2,6 +2,7 @@
 
 ```
 MIT License
+Copyright (c) 2020-present lowlighter
 Copyright (c) 2020-present gh-metrics
```

Comment:
Can you check for any other license lines? I think there's a lot.
Also, the gh-metrics copyright should be 2025-present (inc. elsewhere).

Reply:
> Can you check for any other license lines? I think there's a lot.

@tboy1337, I think that still applies? Did I miss another LICENSE change?